nipy/algorithms/statistics/formula/formulae.py

```diff
@@ -526,7 +527,7 @@ def fromrec(rec, keep=[], drop=[]):
         if keep:
             return np.sum([t for n, t in f.items() if n in keep])
         else:
-            return np.sum(f.values())
+            return np.sum(list(f.values()))
```

Comment:
This a Python 3 thing - I get `dict_values([2, 1])` for numpy 1.8.2, on Python 3.